### ISSUE TITLE
Include "real" page numbers in state.json

### ISSUE THREAD

**Issue by sopoforic** [burtonator/polar-bookshelf#814]
The state.json file for a PDF includes for each page a pageInfo object containing the page number, counting from the beginning of the PDF, e.g.

```
"pageInfo": {
    "num": 230
}
```

However, a PDF may have 'real' page numbers corresponding to those shown on the pages, as for example page number 1 of a book may follow many pages of front matter. I propse an addition to the pageInfo object, such as:

```
"pageInfo": {
    "num": 1,
    "realNum": 1203
}
```

A sample PDF exhibiting this behavior may be found [here](https://www.ams.org/notices/200510/comm-fowler.pdf). The first page of the PDF is page number 1203, which is correctly displayed in Polar's PDF viewer.

**Comment by burtonator** [comment]
yes.. the pageInfo would need a new field and then the UI would need new dropdown to set the 'real' initial page.
